frontend/src/pages/Battle/Battle.tsx

```diff
@@ -11,11 +11,52 @@ import { Button } from "components/Button";
 import { useUnit } from "effector-react";
 import { logsStore } from "model/logs";
 import { battle } from "model/battleLogs";
-// import { userStore } from "model/user";
-// import { isEmpty } from "lodash";
-// import { useAlert } from "@gear-js/react-hooks";
 
 export type BattleProps = {};
+
+type TurnTypes = "move" | "attack" | "miss" | "rest";
+type TurnGeneric<K extends TurnTypes, T> = { [key in K]: T };
+type Turn =
+  | TurnGeneric<"move", { position: number }>
+  | TurnGeneric<"attack", { damage: number; position: number }>
+  | TurnGeneric<"miss", { position: number }>
+  | TurnGeneric<"rest", { position: number }>;
+
+type BattleLogs = {
```

Comment:
now we are using localstorage for battle info logs, so we need re export  this for another file
for example page My Logs (Profile)

Reply:
done